cmd/security-proxy-setup/res/configuration.toml

```diff
@@ -16,8 +16,9 @@
 
 # This is a TOML config file for edgexsecurity service.
 
-[Writable]
 LogLevel = "DEBUG"
+
+[ProxySetup]
```

Comment:
Why was this added?

Reply:
no particular strong reason, i just felt that it can be more structural in this way since potentially you can have many more `RequestTimout` for that naming alone.

Reply:
It is redundant since this is `Proxy Setup's` specific configuration. IMO, the extra level isn't needed. We added the extra levels in the bootstrapper config to make the Env override more readable, but that isn't needed here.

Reply:
fixed.